WordPress/Classes/ViewRelated/NUX/LoginWPComViewController.swift

```diff
@@ -1,6 +1,8 @@
 import UIKit
 
 class LoginWPComViewController: SigninWPComViewController, LoginViewController {
```

Comment:
It would be nice if we could go ahead and loose the inheritance from `SigninWPComViewController` (and ditch a lot of the `overrides` in the process).  All the `Signin` controllers are doomed to deletion... porting the code now will make the clean up pr easier later.